autopush/router/interface.py

```diff
@@ -33,6 +33,13 @@ def __init__(self, status_code=200, response_body="", router_data=None,
         self.router_data = router_data
         self.headers = headers
 
+    def respond(self, handler):
+        handler.set_status(self.status_code)
+        handler.write(self.response_body)
+        for name, val in self.headers.items():
+            handler.set_header(name, val)
+        handler.finish()
+
```

Comment:
Why isn't it sufficient to have the handler take the response and fill that in like it does now?